tools/gen_objcdoc.sh

```diff
@@ -5,9 +5,10 @@
 
 # Generates objc docs for Flutter iOS libraries.
 
-if [[ ! -d "shell/platform/darwin/ios" ]]
+FLUTTER_UMBRELLA_HEADER=$(find ../out -maxdepth 4 -type f -name Flutter.h | grep 'ios_' | head -n 1)
```

Comment:
On the bots, can't we always assume that a particular variant (e.g. `ios_debug_unopt` is present?

Reply:
We could, but I figured this would make it easier to test locally (e.g. I almost never build the debug opt variant, and this way I don't have to edit the file to check it if I'm on ios_debug_unopt)